FirebaseTestProject/Views/PostRow.swift

```diff
@@ -8,25 +8,92 @@
 import SwiftUI
 
 struct PostRow: View {
-    let post: Post
+    @Binding  var post: Post
+    @State var isPresenting: Bool = false
+    @State private var showAlert: Bool = false
+    @State private var systemImage: String = ""
+    let deletePostAction:((Post) -> Void)
+    let isFavoriteAction:((Post) -> Void)
     
+    init(post: Binding<Post>, deletePostAction: @escaping (Post) -> Void, isFavoriteAction: @escaping (Post) -> Void) {
```

Comment:
Update the `PostsList` to use this initializer for `PostRow`. Currently, it doesn’t compile because the `post` parameter is not a binding, and the `deletePostAction` and `isFavoriteAction` are missing.

Reply:
Resolved by [commit 98a62a3563639f771ec93899fab361f63dc747a6](https://github.com/benstone1/InstaCademy/pull/13/commits/98a62a3563639f771ec93899fab361f63dc747a6)